Hmm, this looks like random CI breakage to me: https://ci.centos.org/job/bodhi-pipeline/job/PR-3418/1/execution/node/328/log/

```
12:36:22  Traceback (most recent call last):
12:36:22    File "/usr/bin/diff-cover", line 11, in <module>
12:36:22      load_entry_point('diff-cover==1.0.7', 'console_scripts', 'diff-cover')()
12:36:22    File "/usr/lib/python3.7/site-packages/diff_cover/tool.py", line 340, in main
12:36:22      GitPathTool.set_cwd(directory)
12:36:22    File "/usr/lib/python3.7/site-packages/diff_cover/git_path.py", line 30, in set_cwd
12:36:22      cls._root = cls._git_root()
12:36:22    File "/usr/lib/python3.7/site-packages/diff_cover/git_path.py", line 64, in _git_root
12:36:22      git_root = execute(command)[0]
12:36:22    File "/usr/lib/python3.7/site-packages/diff_cover/command_runner.py", line 44, in execute
12:36:22      raise CommandError(stderr)
12:36:22  diff_cover.command_runner.CommandError: fatal: not a git repository (or any of the parent directories): .git
```

Rescheduling.

--- a/devel/ci/integration/bodhi/Dockerfile-f29
+++ b/devel/ci/integration/bodhi/Dockerfile-f29
@@ -17,6 +17,7 @@ RUN dnf install -y \
     httpd \
     intltool \
     python3-mod_wsgi \
+    python3-pip \
     skopeo \
     /usr/bin/koji
 

--- a/devel/ci/integration/bodhi/Dockerfile-f30
+++ b/devel/ci/integration/bodhi/Dockerfile-f30
@@ -17,6 +17,7 @@ RUN dnf install -y \
     httpd \
     intltool \
     python3-mod_wsgi \
+    python3-pip \
     skopeo \
     /usr/bin/koji
 

--- a/devel/ci/integration/bodhi/Dockerfile-pip
+++ b/devel/ci/integration/bodhi/Dockerfile-pip
@@ -19,6 +19,7 @@ RUN dnf install -y \
     python3-koji \
     /usr/bin/koji \
     python3-mod_wsgi \
+    python3-pip \
     python3-dnf \
     skopeo
 

--- a/devel/ci/integration/bodhi/Dockerfile-rawhide
+++ b/devel/ci/integration/bodhi/Dockerfile-rawhide
@@ -20,6 +20,7 @@ RUN dnf install -y \
     httpd \
     intltool \
     python3-mod_wsgi \
+    python3-pip \
     skopeo \
     /usr/bin/koji
 

